tasks/test.js

```diff
@@ -0,0 +1,39 @@
+module.exports = function(grunt) {
+
+  grunt.config('test', {
+    behat: true,
+    drupal: []
+  });
+
+  grunt.config(['shell', 'testdrupal'], {
+    // @todo Only make this available on Drupal 8+, or change command.
+    // @todo Make configurable.
+    // @todo The phpunit binary (./vendor/bin/phpunit) needs to be executable, GDT changes this somehow during build.
```

Comment:
@jhedstrom This reminded me of the `mode` option on grunt-contrib-copy. #211 should resolve the removal of the executable mode.